### DISCUSSION TITLE
positioning does not take negtive margin or translate into account

### DISCUSSION THREAD

**Discussion by joernroeder** [Akryum/v-tooltip#157] [Q&A]
I'm currently building some map component where i have to center an element of flexible size around a given (absolute) position and therefore the target is a) translated via `transform: translate(-50%, -50%)` or it's old school friend `margin-left: -50%`. but both is not taken into account and tooltip positioning is shifted by `elementWidth / 2`.

**Comment by Akryum** [comment]
Have you tried using offset option?
